docs/plugins/analysis-nori.asciidoc

```diff
@@ -154,6 +154,40 @@ The above `analyze` request returns the following:
 
 <1> This is a compound token that spans two positions (`mixed` mode).
 
+`user_dictionary_rules`::
```

Comment:
For some reason this whole sections doesn't render when I build the docs locally. I played around with it a bit but couldn't get it to work but its probably worth taking another look.

Reply:
Good catch, thanks. I forgot to add the end of section (e.g. `--`) so the whole section was not displayed. I pushed https://github.com/elastic/elasticsearch/pull/36123/commits/adcee291b47a37ca124f5d252cb6f287afe0536c to fix this.